exam/backend/server.js

```diff
@@ -18,9 +18,28 @@ db.sequelize.sync({force:false}).then( () =>{
 
 
 let userRouter = require('./routes/user') ;
+let orgAdminRouter = require('./routes/org_admin') ;
 //app.use('/api', api);
 app.use('/user' , userRouter ) ;
-
-
+app.use('/' , orgAdminRouter ) ;
+let newAgency = db.agency.build( {agency_id:1 , name : "resonance" }) ;
+newAgency.save() ;
```

Comment:
I think Line #25 and #26 are for testing. If yes please remove them, else please help me understand the purpose they serve.